### ISSUE TITLE
[New-UDGrid] Grids show dates as "/Date(...)/"

### ISSUE THREAD

**Issue by kort3x** [adamdriscoll/poshprotools#222]
code:
```
New-UDTable -Title "Events in a table" -Headers @("TimeWritten","Source", "EventID") -Endpoint {
        Get-EventLog -LogName system -Newest 3 -EntryType Error | Out-UDTableData -Property @("TimeWritten","Source", "EventID")
}
    
New-UdGrid -Title "Events in a grid" -Properties @("TimeWritten","Source", "EventID") -Headers @("TimeWritten","Source", "EventID") -Endpoint {
        Get-EventLog -LogName system -Newest 3 -EntryType Error | Out-UDGridData
}
````

output:

![image](https://user-images.githubusercontent.com/13918530/34123041-9311622a-e42e-11e7-961a-b7f75bc25d94.png)

**Comment by adamdriscoll** [comment]
I need to add a -DateTimeFormat parameter to New-UDGrid as well.

**Comment by adamdriscoll** [comment]
Fixed in 1.3.0-beta5
